checker/raw_result.go

```diff
@@ -32,9 +32,45 @@ type RawResults struct {
 	ContributorsResults         ContributorsData
 	MaintainedResults           MaintainedData
 	SignedReleasesResults       SignedReleasesData
+	FuzzingResults              FuzzingData
 	LicenseResults              LicenseData
 }
 
+// FuzzerName represents a fuzzing service.
+type FuzzerName string
+
+const (
+	// FuzzerNameCIFuzz is CIFuzz.
+	FuzzerNameCIFuzz FuzzerName = "CIFuzz"
+	// FuzzerNameOSSFuzz is OSSFuzz.
+	FuzzerNameOSSFuzz FuzzerName = "OSSFuzz"
+	// FuzzerNameGoBuiltin is the built-in Go fuzzer.
+	FuzzerNameGoBuiltin FuzzerName = "GoFuzzer"
+)
+
+// FuzzingData represents different fuzzing done.
+type FuzzingData struct {
+	Fuzzers []Fuzzer
+}
+
+// Fuzzer represent the use of a fuzzer.
+type Fuzzer struct {
+	Name FuzzerName
+	// TODO: WorkflowJob, File, CodeCoverage.
+}
+
+// CodeCoverage represents code coverage.
+type CodeCoverage struct {
+	FilesCoverage []FileCoverage
+	// TODO: aggregate coverage, etc
+}
+
+// FileCoverage represent the coverage of a file.
+type FileCoverage struct {
+	Lines []uint
+	File  File
+}
+
```

Comment:
`CodeCoverage` and `FileCoverage` not used anywhere so maybe remove it in this PR?

Reply:
SG